dpdd.tex

```diff
@@ -828,6 +826,31 @@ \subsubsection{Crowded Field Photometry}
 
 Note that this discussion only pertains to processing of \emph{direct images}. Crowding is not expected to significantly impact the quality of data products derived from \emph{difference images} (i.e., Prompt products).
 
+\subsubsection{Shear Measurement}
+\label{sec:shearMeasurement}
+
+Cosmology is one of the major pillars of LSST science, and a substantial fraction of the survey's cosmological constraining power will come from weak gravitational lensing, which involves estimating the shear field applied to a population of galaxies by foreground matter.
+
+For our pixel-level measurements of galaxy shapes to be usable for shear estimation, the responses of the algorithms we apply to shear must be carefully controlled and/or measured.
+While the state of the art for these algorithms continues to evolve, our plan for at least DR1 and DR2 is to use the \texttt{Metadetection} algorithm of \citet{2020ApJ...902..138S}.
```

Comment:
2020ApJ...902..138S is not rendering in the pdf  version - it probably needs to be added to local.bib

Reply:
Huh; I added that to a ticket branch of `lsst-texmf` and updated the submodule accordingly.  Do we need to put things in `local.bib` until `main` of `lsst-texmf` is updated or something?

Reply:
If the ticket branch of lsst-texmf is not merged then it will not be picked up by the rendered version on lsst.io. Assign me as reviewer if you need a reviewer.  Adding to lsst-texmf  is preferable

Reply:
The submodule is fine but I think the deployment script tries to use the main of lsst-texmf container and might be ignoring the submodule completely. This is something that happened for the early documents.

Reply:
The reference is being picked up now